src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,153 @@
+package duke.task;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * TaskList class is used to store different Task objects and
+ * allows the user to modify the list of tasks through
+ * this class.
+ */
+public class TaskList {
+
+    private List<Task> list;
```

Comment:
The naming would be better set as lists rather than list. It is just so that it is more descriptive